include/AutomationEditor.h

```diff
@@ -114,14 +114,15 @@ public slots:
 protected:
 	typedef AutomationPattern::timeMap timeMap;
 
-	void keyPressEvent(QKeyEvent * ke) override;
-	void leaveEvent(QEvent * e) override;
-	void mousePressEvent(QMouseEvent * mouseEvent) override;
-	void mouseReleaseEvent(QMouseEvent * mouseEvent) override;
-	void mouseMoveEvent(QMouseEvent * mouseEvent) override;
-	void paintEvent(QPaintEvent * pe) override;
-	void resizeEvent(QResizeEvent * re) override;
-	void wheelEvent(QWheelEvent * we) override;
```

Comment:
These incorrect changes (117 - 124) are the result of a merge conflict that was resolved incorrectly, and are corrected in the 5th commit.